app/src/main/java/com/ozonehis/eip/Application.java

```diff
@@ -10,8 +10,10 @@
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.PropertySource;
 
 @Slf4j
+@PropertySource("classpath:eip-client.properties")
 @SpringBootApplication(scanBasePackages = "com.ozonehis.eip.*, ${eip.app.scan.packages}")
```

Comment:
Will this be overridable by `eip.app.scan.packages` property from other class paths/sources @corneliouzbett? Could the below suggestion be good alternative in this case?

```suggestion
@SpringBootApplication(scanBasePackages = "com.ozonehis.eip.*, org.openmrs.eip.*, ${eip.app.scan.packages:}")
```

Reply:
Yes. I had already added `org.openmrs.eip.*` to `eip-client.properties` as not every project will require it. Adding it to main class like that won't overridable anymore, we can only override what we provide in this property `${eip.app.scan.packages}`.